hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/InputPathHandler.java

```diff
@@ -117,9 +124,10 @@ private void parseInputPaths(Path[] inputPaths, List<String> incrementalTables)
     }
   }
 
-  private void tagAsIncrementalOrSnapshot(Path inputPath, String tableName,
+  private void tagAsIncrementalOrSnapshot(Path inputPath, String databaseName, String tableName,
```

Comment:
can change the definition of this method to tagAsIncrementalOrSnapshot(Path inputPath, HoodieTableMetaClient metaClient, List<String> incrementalTables)?
Inside, get the database name and the table name.

Reply:
Yes, but the outer layer also needs databaseName and tableName. I'm not sure which is better